### DISCUSSION TITLE
小米路由器AX3600 Collected errors

### DISCUSSION THREAD

**Discussion by 0xMorty9** [messense/aliyundrive-webdav#267] [Q&A] [answered]
root@XiaoQiang:/userdisk# opkg install aliyundrive-webdav_1.2.0_aarch64_cortex-a
53.ipk 
Collected errors:
 * pkg_init_from_file: Malformed package file aliyundrive-webdav_1.2.0_aarch64_cortex-a53.ipk.
小米路由器AX3600

这个是架构输出 
root@XiaoQiang:/userdisk# opkg print-architecture
arch all 1
arch noarch 1
arch aarch64_cortex-a53 10
arch aarch64_generic 100

**Comment by messense** [comment, marked as answer]
不支持小米路由默认系统，请刷 OpenWrt.
